### PR TITLE
Support for negative numbers in computed bindings

### LINKED ISSUE #2264: Negative numbers in computed bindings
```javascript
Polymer(
...
    properties : {
        negative_num : {type: Number, value: -1},
    },
    some_method : function (num) {
        return num;
    }
....

```
Not working, empty inside:
```html
<span>[[some_method(-1)]]</span>
```
Working, `-1` text inside:
```html
<span>[[some_method('-1')]]</span>
```
Working, `-1` text inside:
```html
<span>[[some_method(negative_num)]]</span>
```

### PR DESCRIPTION
**Pull request by nazar-pc**
Fixes #2264

### CODE REVIEW

**Comment by azakus** [comment]
Can you add a test with invalid data, such as `{{computeNegativeNumber(-A)}}` ?

**Comment by nazar-pc** [comment]
Do you mean test as test or as additional check? I'm not sure how it should behave in such case.

File: src/standard/effectBuilder.html [hunk 1]
```diff
@@ -221,6 +221,7 @@
           a.literal = true;
           break;
         case '#':
+        case '-':
```
Comment (azakus) [inline src/standard/effectBuilder.html]: I think we should check this far earlier, and then check that the next character is a number literal.

Something like

```js
var fc = arg[0];
if (fc === '-') {
  fc = arg[1];
}
if (fc >= '0' && fc <= '9') {
```

**Comment by azakus** [comment]
@nazar-pc That is a great question. It could either be a `NaN` or use '-A' as a literal. @sorvell, do you have a preference?

**Comment by nazar-pc** [comment]
I wanted to make patch as simple as possible.
Moving check earlier as you suggested naturally results in `undefined` for `{{computeNegativeNumber(-A)}}`, which in my opinion is reasonable.

**Comment by azakus** [comment]
LGTM!

**Pull request merged**